src/js/mediaelement-i18n-locale-pl.js

```diff
@@ -65,80 +65,80 @@
 
 			// mep-player
 			"mejs.video-player": "Odtwarzacz wideo",
-			"mejs.audio-player": "Odtwarzacz audio"
+			"mejs.audio-player": "Odtwarzacz audio",
 
 			// mediaelementplayer-feature-ads
-			//"mejs.ad-skip": "Skip ad",
-			//"mejs.ad-skip-info": "Skip in %1 second(s)",
+			"mejs.ad-skip": "Pomiń reklamę",
+			"mejs.ad-skip-info": "Pomiń za %1 sek.",
```

Comment:
According to the rules of plurals, it should be 3 translations, and I only see one, can you add the other 2 please?